server/src/main/java/com/linecorp/centraldogma/server/internal/ArmeriaJacksonObjectMapperProvider.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.server.internal;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.linecorp.armeria.common.JacksonObjectMapperProvider;
+
+public final class ArmeriaJacksonObjectMapperProvider implements JacksonObjectMapperProvider {
+
+    @Override
```

Comment:
I don't see that a custom `JacksonObjectMapperProvider` with SPI is a good idea for a module that can be used as a library. Unfortunately, the scope of `ArmeriaJacksonObjectMapperProvider` is not limited to Central Dogma. `ArmeriaJacksonObjectMapperProvider` may override user's `JacksonObjectMapperProvider`.
What do you think of creating a custom `Jackson{Request,Response}ConverterFunction` instead? It needs more changes though.

Reply:
I thought it was okay because this class is in the server module and users do not have the server module in their environment.
However, they can have a problem because this can be used in the test. 😓  Let me fix that. Thanks!